src/main/java/com/zemnov/salon/controller/OrderAdminController.java

```diff
@@ -0,0 +1,63 @@
+package com.zemnov.salon.controller;
+
+import com.zemnov.salon.model.Master;
+import com.zemnov.salon.model.Order;
+import com.zemnov.salon.service.OrderAdminService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Controller
+@RequestMapping("/orderAdmin")
+@PreAuthorize("hasAuthority('ADMIN')")
+public class OrderAdminController {
+
+    @Autowired
+    private OrderAdminService orderAdminService;
+
+    @GetMapping
+    public String editOrders(Model model){
+
+        List<Order> orders=orderAdminService.findAllOrders();
+        model.addAttribute("orders", orders);
+
+        return("orderAdminPage");
+    }
+
+    @GetMapping("{order}/edit")
+    public String orderAdminEditForm(@PathVariable Order order, Model model) {
+
+        List<Master> masters = orderAdminService.orderAdminEditForm(order);
+
+        model.addAttribute("masters", masters);
+        model.addAttribute("order", order);
+
+        return "orderAdminEdit";
+    }
+
+    @GetMapping("{order}/delete")
+    public String orderAdminDeleteForm(@PathVariable Order order, Model model) {
+        model.addAttribute("order", order);
```

Comment:
As i can see, you leave first line in method block empty, so let's respect this style in all blocks